sdk/data/azcosmos/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos
+
+go 1.16
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.16.2
```

Comment:
This is an old version of `azcore`.  We did a massive refactor of it.  Can you please move to `v0.19.0`?  This will get you current with all the other releases.

Reply:
Done